bin/riemann-aws-rds-status

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+require 'rubygems'
+require 'fog'
+require 'date'
+require 'time'
+require 'json'
+
+require File.expand_path('../../lib/riemann/tools', __FILE__)
+
+$0 = __FILE__ # Let's not expose our AWS keys in the process list
+
+class Riemann::Tools::AWS
+  include Riemann::Tools
+
+  opt :access_key, "AWS access key", :type => String
+  opt :secret_key, "Secret access key", :type => String
+  opt :region, "AWS region", :type => String, :default => 'eu-west-1'
+  opt :dbinstance_identifier, "DBInstanceIdentifier", :type => String
```

Comment:
Should there be a default here? Won't you get a .metric_name at https://github.com/aphyr/riemann-tools/pull/109/files#diff-91de02aa50aadb39fa72dbbc2344eae3R35 ?